<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MagnetGeneratorPrefWidget</class>
 <widget class="QWidget" name="MagnetGeneratorPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>317</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Magnet Generator Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_x3">
   <item>
    <widget class="QGroupBox" name="groupBox_x2">
     <property name="title">
      <string>Magnet Generator</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_x2">
      <item>
       <layout class="QGridLayout">
        <item row="1" column="0">
         <widget class="QCheckBox" name="kcfg_torrenttracker">
          <property name="toolTip">
           <string>Grabs a tracker from the torrent file</string>
          </property>
          <property name="text">
           <string>Add a tracker from the torrent file</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="kcfg_customtracker">
          <property name="toolTip">
           <string>Define what tracker to add</string>
          </property>
          <property name="text">
           <string>Add a custom tracker URL</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="kcfg_tr">
          <property name="toolTip">
           <string>The tracker URL that will be a part of the magnet link</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="kcfg_dn">
          <property name="toolTip">
           <string>Add the torrent name in the URI</string>
          </property>
          <property name="text">
           <string>Add name</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="kcfg_onlypublic">
          <property name="toolTip">
           <string>Only enable on public torrents</string>
          </property>
          <property name="text">
           <string>Disable the menu item for private torrents</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="kcfg_popup">
          <property name="toolTip">
           <string>Give feedback in form of a popup</string>
          </property>
          <property name="text">
           <string>Show popup</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
